packages/legend-studio/src/components/shared/Icon.tsx

```diff
@@ -84,6 +86,19 @@ export const DatabaseIcon: React.FC = () => (
     <FaDatabase />
   </div>
 );
+
+export const SchemaIcon: React.FC = () => (
+  <div className="icon icon--database color--schema">
```

Comment:
Why are we reusing the style like this `icon--database`, can we add a new style?